.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: "Build Package"
-        uses: pyansys/pydpf-actions/build_package@v2.1
+        uses: pyansys/pydpf-actions/build_package@v2.2.dev1
```

Comment:
@PProfizi why do we need to use 3 different versions of pydpf-actions here?

Reply:
@cbellot000 Indeed, I was bumping all to 2.2.dev1 in core, was going to do the same here, now it is done.